### DISCUSSION TITLE
Meaning of negative expression levels in Violin Plot

### DISCUSSION THREAD

**Discussion by thugib** [satijalab/seurat#4969] [Q&A]
I used the VlnPlot on a Seurat object that has been clustered with UMAP. The VlnPlot is using split.by, so there is a plot for two conditions, and in this case, for each of 11 clusters. The specific command is: VlnPlot(root230.combined.sct.res03, features = c("AT4G18350"), split.by = "treatment")
Some of the violin plots cover a range of both positive and negative expression levels.  I have put a .png image of a typical result below.  
![image](https://user-images.githubusercontent.com/253391/130299329-16dedf1f-dbda-4f81-8e34-6cab6e039773.png)

   From the image you can see that while most of the violins have only cells with either positive expression values or 0, some of the violins have a large range of cells in which the expression level of the gene of interest can be positive or negative. 
  Because expression level is relative, I was wondering what the comparison was between that determines the expression level in this plot ? Is it the expression level of the gene as compared the gene's expression level across the whole data set, or the expression level compared to the gene's expression level in the particular cluster, or expression level compared to expression of cells in he particular condition, in this case plus or minus ?  What accounts for negative expression values ?

**Comment by timoast** [comment]
Are you plotting integrated data here? If so that would explain the negative values. We recommend using the un-integrated data fro visualization and for differential expression

**Reply by thugib** [reply]
Yes, I used SCTransform on the data. So, can I plot un-integrated data from the Seurat object that has the SCT data ? I also thought that the default for slot in the VlnPlot was "data" ?
